app/browser/rightClickMenuWithSpellcheck.js

```diff
@@ -75,7 +75,6 @@ var simpleChecker = window.spellChecker = {
 
 webFrame.setSpellCheckProvider(
 	appLocale,
-	true,
```

Comment:
are we disabling the spellchecker?

Reply:
@IsmaelMartinez , I had upgraded the all the module versions. As per the latest version, it requires only 2 parameters. The second one is 3rd param in the old version.